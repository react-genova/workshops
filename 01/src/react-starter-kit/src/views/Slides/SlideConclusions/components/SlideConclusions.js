//@flow
import React from 'react';
import styled from 'styled-components';

import CardConclusions from './CardConclusions';

const Container = styled.div`
    display: flex;
    flex-direction: row;
    flex-wrap: wrap;
    justify-content:space-evenly;
    align-items: center;
    align-content: center;
`;

const SlideConclusions = () => (
    <Container>
        <CardConclusions />
    </Container>
);

export default SlideConclusions;