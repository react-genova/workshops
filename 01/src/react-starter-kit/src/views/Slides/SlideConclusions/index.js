//@flow
export {default} from './components/SlideConclusions';