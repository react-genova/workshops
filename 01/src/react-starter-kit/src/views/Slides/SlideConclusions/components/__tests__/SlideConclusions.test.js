//@flow
import React from 'react';
import { shallow } from 'enzyme';
import SlideConclusions from '../SlideConclusions';

describe('Slide Conclusions', () => {

    it('snapshot check', () => {
        const component = shallow(<SlideConclusions />);
        expect(component).toMatchSnapshot();
    });

});