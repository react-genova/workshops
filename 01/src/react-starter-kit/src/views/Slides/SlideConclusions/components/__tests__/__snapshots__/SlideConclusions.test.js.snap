// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Slide Conclusions snapshot check 1`] = `
<styled.div>
  <CardConclusions />
</styled.div>
`;
